ledger/src/blockstore_processor.rs

```diff
@@ -2129,6 +2206,19 @@ fn load_frozen_forks(
                 all_banks.retain(|_, bank| bank.ancestors.contains_key(&root));
                 m.stop();
                 root_retain_us += m.as_us();
+
+                // If this root bank activated the feature flag, update migration status
+                if migration_status.is_pre_feature_activation() {
```

Comment:
startup equivalent of https://github.com/anza-xyz/alpenglow/blob/d6f08a5b9126aec839e462a76a0cd72cdda649ea/core/src/replay_stage.rs#L2843-L2844